src/_scss/components/_titleBarFilter.scss

```diff
@@ -2,6 +2,12 @@
     display: flex;
     width: auto;
     align-items: center;
+
+    transform: translateY(2px);
```

Comment:
This shouldn't be necessary to position the element in its parent.
I think if you remove this and put height: 48px on both the subawrd-dropdown__container and its child with classnames 'filter__dropdown-container subaward-dropdown__wrapper' it should be aligned correctly.
The 48px comes from the stickyHeader total height of 60 minus its padding of 6 top and bottom.

Reply:
did you actually try this or are you guessing?

Reply:
I did it in devtools, I didn't make a new css file for it